vocab/credentials/v2/vocabulary.yml

```diff
@@ -33,6 +33,11 @@ class:
     label: Refresh service
     comment: A Refresh Service is a mechanism that can be utilized by software agents to retrieve an updated copy of a Verifiable Credential.
 
+  - id: Credential
+    label: Credential
+    comment: |
+      A Credential is a set of one or more claims made by an issuer.
```

Comment:
Should "claims" be defined in this vocabulary?

One definition exists already in https://www.w3.org/TR/vc-data-model-2.0/#dfn-claims, and a longer explanation is also available in https://www.w3.org/TR/vc-data-model-2.0/#claims

Reply:
Makes sense to me. Created https://github.com/w3c/vc-data-model/issues/1061